react-components/src/components/RuleBasedOutputs/RuleBasedOutputsSelector.tsx

```diff
@@ -26,24 +28,17 @@ export function RuleBasedOutputsSelector({ ruleSet }: ColorOverlayProps): ReactE
     fetchNextPage
   } = useAllMappedEquipmentAssetMappings(models);
 
-  // clean up the appearance
-  models.forEach((model) => {
-    const currentModel = model as CogniteCadModel;
-    currentModel.removeAllStyledNodeCollections();
-
-    currentModel.setDefaultNodeAppearance({
-      color: new Color('#efefef')
-    });
-  });
-
   useEffect(() => {
-    if (!isFetching && hasNextPage !== undefined) {
+    if (!isFetching && hasNextPage === true) {
       void fetchNextPage();
```

Comment:
Hmm right. Make sense. So for that we have to change the SearchHooks story as well? Ofc not related to this but assuming that they were using the similar approach.